tests/services/test_projects_service.py

```diff
@@ -49,7 +49,16 @@ def test_create_project():
     projects_service.instrument_service.fetch_instruments.return_value = [default_instrument()]  # type: ignore
 
     # When
-    project_new = ProjectNew(name=project_name, instruments=[])
+    project_new = ProjectNew(
+        name=project_name,
+        instruments=[],
+        type="project_type",
+        open_source="project_open_source",
+        publication_category="project_publication_category",
```

Comment:
Wouldn't this be more logical?

```suggestion
    project_new = ProjectNew(
        name=project_name,
        instruments=[],
        ai_act_profile={
            "type": "project_type",
            "open_source": "project_open_source",
            "publication_category": "project_publication_category",
            ...
```

Reply:
This object is used to parse the data that is filled in the create project form. It matches on keys from the form. That's why it's implemented like this now. Changing it to your suggestion would be possible I think, but we need to change the way to form passes the data.

Reply:
Ok, sounds this is closer to the form than I'd realized. Not neccessary for me.